jepler_udecimal/utrig.py

```diff
@@ -218,10 +224,15 @@ def acos(x, context=None):
 
     with localcontext(context) as ctx:
         ctx.prec += 2
-        r = atan((1 - x * x).sqrt() / x)
+        if x == 1:
+            r = Decimal(0). # 0 radians
```

Comment:
Looks like a stray dot here? Or am I missing something?

Reply:
Yes, stray dot. Operator error. The GitHub editor inserted a period at the end of a sentence after two spaces are typed.